espnet/nets/pytorch_backend/transformer/encoder.py

```diff
@@ -187,3 +205,145 @@ def forward_one_step(self, xs, masks, cache=None):
         if self.normalize_before:
             xs = self.after_norm(xs)
         return xs, masks, new_cache
+
+
+class EncoderMix(Encoder, torch.nn.Module):
```

Comment:
We should place this class in a new module, for example `encoder_mix.py`.

Reply:
Done. Thanks!